.github/workflows/docker-ci.yml

```
@@ -0,0 +1,52 @@
+name: Docker CI
+
+on:
+  push:
+    branches:
+      - 'chore-add-image-build-test-to-github-actions'
```

The `on` key hasn't been updated

```suggestion
on:
  push:
    branches:
      - 'main'
  pull_request:
```

---

roo I forgot it ! I'm fixing that

---

The `pull_request` trigger is still missing